jest.config.js

```diff
@@ -1,3 +1,11 @@
 module.exports = {
   testMatch: ['**/tests/**/*.js'],
+  coverageThreshold: {
```

Comment:
Didn't know that this already prevents CI from continuing as it makes jest fails if threshold is not met!

Reply:
Jest is awesome for that. Just set a coverage threshold and you're good to go as long as you run `jest --coverage`!